components/brave_rewards/browser/publisher_info_database_unittest.cc

```diff
@@ -885,7 +886,7 @@ TEST_F(PublisherInfoDatabaseTest, GetActivityList) {
   filter_4.excluded = ledger::EXCLUDE_FILTER::FILTER_ALL;
   EXPECT_TRUE(publisher_info_database_->GetActivityList(0,
                                                         0,
-                                                        filter_4,
+                                                        filter_4.Clone(),
```

Comment:
`filter_4.Clone()` should use `std::move(filter_4)` as `filter_4` is no longer used

Reply:
fixed